### ISSUE TITLE
Initial admin creation not possible

### ISSUE THREAD

**Issue by Schmandre** [polonel/trudesk#449]
**Is this a BUG REPORT or FEATURE REQUEST?**:

- [x] BUG 
- [ ] FEATURE

**What happened**:
After I try to create a admin user during setup process I'll get this error:
```
{"success":false,"error":{"errors":{"value":{"name":"ValidatorError","message":"Path `value` is required.","properties":{"message":"Path `value` is required.","type":"required","path":"value"},"kind":"required","path":"value"}},"_message":"settings validation failed","name":"ValidationError","message":"settings validation failed: value: Path `value` is required."}}
```

If I retry after that with the same input, I'll get this error message:
```

{"success":false,"error":{"driver":true,"name":"MongoError","index":0,"code":11000,"keyPattern":{"name":1},"keyValue":{"name":"gen:version"}}}
```

**What did you expect to happen**:
Working admin profile

**How to reproduce it (as minimally and precisely as possible)**:
1. Create a mongodb and trudesk container
2. Go to webinterface
3. Fill in all informations
4. Submit
5. Error

**Anything else we need to know?**:
Using a seperate mongodb and trudesk container, caused by Rancher environment

**Environment**:
- Trudesk Version: Trudesk v1.1.10
- OS (e.g. from /etc/os-release): NAME="Ubuntu" VERSION="20.04.3 LTS (Focal Fossa)"
- Node.JS Version: (docker)
- MongoDB Version: 4.0
- Is this hosted on cloud.trudesk.io: No

**Comment by polonel** [comment]
Have you tried with the codebase install and not docker. I'm trying to isolate the issue if it's a docker-related issue only. 

I'm not seeing it with my test VM.

**Comment by Schmandre** [comment]
> Have you tried with the codebase install and not docker? I'm trying to isolate the issue if it's a docker-related issue only.
> 
> I'm not seeing it with my test VM.

No, tried it just with docker atm.

**Comment by JesusFregoso** [comment]
you can solve it doing this #446

**Comment by JesusFregoso** [comment]
the mongo error is because you already try to create an installation.

enter into docker mongo bash and delete the trudesk database and start over from fresh install.

`docker exec -it mongodb mongo`

then inside mongo run this commands.
list databases to verify trudesk database

`db.adminCommand( { listDatabases: 1 } )`

then use trudesk

`use trudesk;`

then drop the database

`db.dropDatabase();`

start over with trudesk

**Comment by polonel** [comment]
This was fixed in [v1.1.11](https://github.com/polonel/trudesk/releases/tag/v1.1.11)
